extensions/amp-animation/0.1/web-animation-types.js

```diff
@@ -184,6 +184,7 @@ export const WebAnimationTimingFill = {
 const WHITELISTED_RPOPS = {
   'opacity': true,
   'transform': true,
+  'transform-origin': true,
```

Comment:
hey @aghassemi, question: if I would like to whitelist this property, do I have to do something on the validator side as well?

Reply:
No need for validator changes. It is all runtime validation